index.html

```diff
@@ -14,7 +14,39 @@
   <entry class="blue">
     <img class="header" src="assets/img/decim11.jpg" />
     <h2>Comparing Triangle-Collapse and Edge-Collapse</h2>
-    <p>Triangle-Collapse may be a terrific Mesh Decimation methedology for some scenarios, but for our specific requirements (Terrain LODs with certain immutable vertices at the edge) it isn't optimal as its caveats lie exactly in the areas that hurt the most here.</p>
+    <p>As Vertex-Elimination is too limited to really produce any interesting results with complex meshes, as it's decimation floor is relatively high for any given mesh, let's only have a look at how Edge-Collapse & Triangle-Collapse fair on some more complex models and what their characteristics are when pushed to their limits.</p>
```

Comment:
- `As Vertex-Elimination ... as it's decimation floor` -> as dopplung, nicht so stoerend
- `fair` einzige bedeutung als verb auf dicct.cc: `abschrägen`